qiime2/core/testing/plugin.py

```diff
@@ -100,33 +100,76 @@
     SingleIntFormat, RedundantSingleIntDirectoryFormat,
     citations=[citations['mayer2012walking']])
 
-dummy_plugin.register_semantic_type_to_format(
+
+# Create IntSequence1 import example usage example
+def is1_use(use):
+    def factory():
+        from qiime2.core.testing.format import IntSequenceFormat
+        from qiime2.plugin.util import transform
+        ff = transform([1, 2, 3], to_type=IntSequenceFormat)
+
+        ff.validate()
+        return ff
+
+    to_import = use.init_format('to_import', factory, ext='.hello')
+
+    ints = use.import_from_format('ints',
```

Comment:
The linter is upset about these, you can delete the variable assignment and the usage example won't care